manager/controllers/common_test.go

```diff
@@ -0,0 +1,72 @@
+package controllers
+
+import (
+	"app/base/core"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+)
+
+// Content type variables
+var contentTypeCSV = "text/csv"
+var contentTypeJSON = "application/json"
+
+// Init request
+func PrepareRequest(method string, url string, body io.Reader) (w *httptest.ResponseRecorder, req *http.Request) {
+	req, _ = http.NewRequest(method, url, body)
+	return httptest.NewRecorder(), req
+}
+
+// Check if additional header
+func CheckHeader(req *http.Request, contentType *string) {
+	if contentType != nil {
+		contentTypeVal := *contentType
+		req.Header.Add("Accept", contentTypeVal)
+	}
+}
```

Comment:
I have a few suggestions. 

- it doesn't need to be exported (if I didn't miss that), so it can have `lowerCase` name
- `CheckHeader` is a little misleading name since it does not actually checks anything. It adds `Accept` header. Something like `acceptContentType` or `addAcceptHeader` would be more clear in my opinion
- `contentType` can be `string` it is not neccessary to use `*string`, you will just check for `""` instead of `nil`. It will simplify some calls to `CreateRequest` where you would not need to define a variable holding content type value like this:
```go
// i've seen something like this in some test above
ct := "text/html"
CreateRequest(arg1, arg2, etc, &ct)
// you could call it as if you used pointer
CreateRequest(arg1, arg2, etc, "text/html")
```

```suggestion
func (req *http.Request) acceptContentType(ct string) {
	if ct != "" {
		req.Header.Add("Accept", ct)
	}
}
```

Reply:
You are right, this looks way better and cleaner. I will change it.